packages/postcss-normalize-string/src/index.js

```diff
@@ -248,22 +252,16 @@ function pluginCreator(opts) {
       const cache = new Map();
 
       css.walk((node) => {
```

Comment:
The code picks a property name based on the node type, then uses the property name to select a function and the property to update on the node. I find using a map is not necessary since the node types are fixed